docs/DeveloperGuide.md

```diff
@@ -411,6 +411,100 @@ The following activity diagram summarizes what happens when a user executes a ne
+
+The following sequence diagram illustrates how a `show` operation goes through the `Logic` component and sets the Patient Note Display. 
+
+<puml src="diagrams/ShowSequenceDiagram.puml" alt="FindSequenceDiagram" />
+
+<box type="info" seamless>
+
+**Note:** The lifeline for `ShowCommand` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
+
+</box>
+
+The following activity diagram summarizes what happens when a user executes a new command:
+<div style="text-align: center;">
+    <puml src="diagrams/ShowCommandActivityDiagram.puml" width="250"/>
+</div>
+
+#### Design Considerations & Alternatives Considered 
+
+**Aspect: How to show the notes of patient on Patient Note Display.**
+
+- **Alternative 1 (Current Choice):** Show patient's notes on one command using `IC_NUMBER`.
+
+    - Pros: Users able to view specific user's notes.
+
+    - Cons: Only allow for one parameter which is `IC_NUMBER`.
+
+- **Alternative 2:** Only show the patient notes which was last searched for.
+    - Pros: Easy to implement.
+
+    - Cons: Brings inconvenient to users if they only want to view a specific user's notes.
+
+**Aspect: How to clear the notes of patient on Patient Note Display.**
+
+- **Alternative 1 (Current Choice):** Clears patient's notes on Patient Note Display using one command.
+    - Pros: Easy to implement. Users do not need to remember so many commands.
+
+    - Cons: The command might not be intuitive.
+
+- **Alternative 2:** Clears patient's notes on Patient Note Display with another command.
+
+    - Pros: The command is more intuitive.
+
+    - Cons: Users are required to know an extra command, which reduce usability of application.
+
+### NoteDisplay (UI Component)
+
+The `NoteDisplay` allows the users to view the notes of the selected patient in the patient list. All notes in the `Person Card`.
+
+#### Implementation 
+
+`NoteDisplay` inherits `UIPart` and is used to display the notes section of the patient details. More details of the class implementation can be seen in the class diagram below.
+
+
+`NoteDisplay` has a private field `noteDisplay` which is of type `TextArea`. The `NoteDisplay` has a method called `setNoteToUser` which takes in a string input and changes the `noteDisplay` through its `settext` method. 
+The notes will then be display in a section which is a FXML `VBox`. 
+
+#### Design Consideration & Alternatives Considered 
+
+**Aspect: How the notes of the patient are displayed**
+
+- **Alternative 1 (Current Choice):** Display the notes of the patients in a separate panel.
+    - Pros: The information of patient is more well organised. It allows users to have a more comprehensive view of their patient notes.
+  
+    - Cons: One additional command is needed to view the notes of the patients.
+
+- **Alternative 2:** Display the notes of the patients in the `PersonalListCard` with the rest the of the details.
```

Comment:
do u mean person card or person list panel? 
we can standardise whether we want to code block for it as we didn't code block for patient list panel above :)

Reply:
okok